### PR TITLE
Update for Xcode 10.1 final release, including last minute Pen/Tap API

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

File: src/uikit.cs [hunk 1]
```diff
@@ -18433,4 +18433,45 @@ interface UIFontMetrics {
 #endif // !WATCH
 	}
 
+#if !WATCH
```
Comment (VincentDondain) [inline src/uikit.cs]: Minor but could we not have avoided that with empty skeletons like we usually do?

Comment (spouliot) [reply src/uikit.cs]: yes, but the errors were on an existing type and I did not want to PR to touch unrelated code.

**Comment by spouliot** [comment]
build

**Comment by spouliot** [comment]
* watchOS - simulator (Crashed; Succeeded; Ignored) -> HE0038 known random issue -> 
https://github.com/xamarin/maccore/issues/581

**Comment by spouliot** [comment]
Most of the failures are now also reported on `master` - which is to be expected since they are related to the bot upgrade to 10.14. I'm merging this one to unblock other release tasks.

**Pull request merged**